ee/vellum_ee/workflows/display/nodes/base_node_display.py

```diff
@@ -404,7 +404,11 @@ def serialize_value(self, display_context: "WorkflowDisplayContext", value: Base
             }
 
         if isinstance(value, ParseJsonExpression):
-            raise ValueError("ParseJsonExpression is not supported in the UI")
+            json_string = self.serialize_value(display_context, value._expression)
+            return {
+                "type": "PARSE_JSON",
+                "json_string": json_string,
```

Comment:
[blocking] as discussed, we'll want to model this as a `"UNARY_EXPRESSION"` so that the frontend can you the same primitives it's using to represent things like `notNull`. both are functions from `(T) -> O`, where `parse_json` is `String -> JSON` and `notNull` is `Any -> Boolean`